cmd/envcd/main.go

```diff
@@ -29,6 +29,11 @@ import (
 func main() {
 	configFile := flag.String("config", "config/envcd.yaml", "envcd -config config/envcd.yaml")
 	flag.Parse()
+	configData := config.NewConfig(configFile)
+	// show start information & parser config
+	configData.StartInformation()
+	service := openservice.InitService(envcd.Start(configData.ExchangerConnMetadata),
```

Comment:
rename to `envcdService` ?

Reply:
This should be a separate instance and should not appear in the API?